src/io/perun.clj

```diff
@@ -26,6 +26,27 @@
   '[[org.pegdown/pegdown "1.6.0"]
     [circleci/clj-yaml "0.5.3"]])
 
+(deftask dump-meta
```

Comment:
Yes it is. It's probably not ideal yet but in general it probably wont hurt to provide something along those lines. :) I just pushed another commit with another `map-fn` option that turned out to be more useful.